script.js

```diff
@@ -0,0 +1,99 @@
+window.onload = function () {
+	startLoading();
+	getData('https://code-your-future.github.io/api-demo/area/index.json',logAreas);
+};
+function getData(url,displaycallback) {
+	var jsonString;
+	var http = new XMLHttpRequest();
+	// Specify the Method and the URL we want to access
+	http.open('GET', url);
+	// Function to be called as the request happens
+	http.onreadystatechange = function() {
+		// readyState === 4 means the request has finished (http://www.w3schools.com/xml/ajax_xmlhttprequest_response.asp)
+		// status === 200  means the request was OK
+		if (this.readyState === 4 && this.status === 200) {
+			displaycallback(this.responseText);
+			loadingFinished();
+		};
+	};
+	http.send(); // Make the request
+};
+function logAreas(jsonString) {
+	var myObject = JSON.parse(jsonString);
+	for (var key in myObject) {
+		for (var i in myObject[key]) {
+			createBtn([myObject[key][i]]);
+		};
+	};
+	addingEvents();
+};
+function logAreaInfo(jsonString) {
+	var myObject = JSON.parse(jsonString);
+	var counter = 0;
+	for (var i in myObject['data']) {
+		createDiv('info');
+		for (var j in myObject['data'][i]) {
+			document.getElementsByClassName('info')[counter].innerHTML += ('<p>' + '<strong>' + [j] + ':  ' + '</strong>'  + myObject['data'][i][j] + '</p>' );
+		};
+		counter++;
+	};
+};
+function createBtn(textNode) {
+	var btn = document.createElement("BUTTON");
+	var text = document.createTextNode(textNode);
+	btn.appendChild(text);
+	var attr = document.createAttribute("class")
+	attr.value = "btnArea";
+	btn.setAttributeNode(attr);
+	attr = document.createAttribute("type")
+	attr.value = "button";
+	btn.setAttributeNode(attr);
+	document.getElementById("buttons").appendChild(btn);
+	// var x = document.getElementById("container");
```

Comment:
In Git (and version control in general) you can view old code by looking at previous commits, so you don't need to keep old code by comments